test/Microsoft.AspNet.Mvc.FunctionalTests/Compiler/Resources/ViewEngineController.ViewWithPaths.txt

```diff
@@ -0,0 +1,21 @@
+﻿<Layout>
+    /Views/ViewWithPaths/_Layout.cshtml
+    /Views/ViewWithPaths/Index.cshtml
+</Layout>
+
+<ViewStart>
+    Views\ViewWithPaths\_ViewStart.cshtml
```

Comment:
agree a separate issue might come of this but I wouldn't file it unless we get complaints.  it's an existing problem that's just made a bit more visible here.